common/app/common/dfp/PageSkinSponsorship.scala

```diff
@@ -3,15 +3,22 @@ package common.dfp
 import common.{Edition, Logging}
 import play.api.libs.json._
 
+/** A PageSkinSponsorship
+  *   is a special decoration of a DFP LineItem that we need to scan for ourselves,
+  *   because DFP doesn't have this concept.
+ */
 case class PageSkinSponsorship(lineItemName: String,
                                lineItemId: Long,
                                adUnits: Seq[String],
+                              // Targeting properties
                                editions: Seq[Edition],
                                countries: Seq[String],
-                               isR2Only: Boolean,
+                               isR2Only: Boolean, // Legacy platform.
```

Comment:
This is redundant now.  But probably to be removed in a separate PR.

Reply:
👍 Filed a card to get rid of it: https://trello.com/c/EHgmnw1i/142-get-rid-of-isr2only-in-pageskin-sponsorship